picky-asn1-x509/src/private_key_info.rs

```diff
@@ -329,6 +339,16 @@ impl RsaPrivateKey {
         &self.coefficient
     }
 
+    #[cfg(feature = "zeroize")]
+    #[deprecated(note = "field is now public")]
+    pub fn into_public_components(mut self) -> (IntegerAsn1, IntegerAsn1) {
+        (
+            std::mem::take(&mut self.modulus),
+            std::mem::take(&mut self.public_exponent),
+        )
+    }
+
+    #[cfg(not(feature = "zeroize"))]
```

Comment:
I think it’s best to just make this the default implementation even if `zeroize` feature is not enabled:
- Less code
- The empty "default" `Vec` is optimized and doesn’t trigger memory allocation (when created through `Vec::new` or `Vec::default` as seen [here](https://github.com/rust-lang/rust/blob/481db40311cdd241ae4d33f34f2f75732e44d8e8/library/alloc/src/vec/mod.rs#L424) and [here](https://github.com/rust-lang/rust/blob/f1f721e64014863f41c1a386b04af04c2de25321/library/alloc/src/raw_vec.rs#L64))
- I’m pretty sure this produce the same code in release profile anyway

Reply:
Same in picky/src/key.rs.